src/Microsoft.Health.Fhir.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -53,7 +54,7 @@ public override void OnActionExecuted(ActionExecutedContext context)
                     new OperationOutcome
```

Comment:
We probably should avoid using OperationOutcome and have our own class? Or is that part of the refactor for the moving the APIs to STU3 assembly?

Reply:
Yep, splitting API will be different PR